shipyard_blueprints/box/shipyard_box/cli/download.py

```diff
@@ -207,41 +194,53 @@ def get_client(service_account):
         client.user().get()
         return client
     except BoxOAuthException as e:
-        print(
-            f"Error accessing Box account with pervice account "
```

Comment:
there was a typo and non of the values in 
            f"Error accessing Box account with pervice account "
            f"developer_token={developer_token}; client_id={client_id}; "
            f"client_secret={client_secret}"
were declared anywhere else